.storybook/welcome.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import Swatch from './Swatch/'
+
+storiesOf("Welcome", module).add("Introduction", () => (
```

Comment:
What is this one for?

Reply:
It'll be the "homepage" of the design guide.

More to come will be:
- colour swatches (implemented)
- logos for third parties / internal staff
- how our grid works, and supported browser media-query sizes (mobile, desktop, and large desktop size ranges)
- typography (I'm thinking of a standardised set of sizes, and we can look at TC's codebase and remove some of the misc sizing across elements)
- browser compatibility (what we decided as a team from @nickbrowne's browser stats work)
- probably a guide on how to use the components (ala http://feelix.myob.com/components/getting-started)

Thoughts?

Reply:
Maybe I've misunderstood, but I thought we were seriously considering dropping our custom grid (and more) for something pre-existing that we can make use of.

The vibe I'm getting with this work is that we're instead going to be persisting with maintaining all of it instead of cutting it loose.

Reply:
Sounds good. Maybe for now we could just have something on that page so it's not blank.

"Welcome to the TC design guide. It contains examples of the React components we use in our apps."

...or words to that effect 😄

Reply:
Added, good call.

![window capture 5](https://user-images.githubusercontent.com/127084/40633076-7939e184-6330-11e8-854e-57830596b065.png)